okio-files/src/commonMain/kotlin/okio/Filesystem.kt

```diff
@@ -120,6 +120,9 @@ abstract class Filesystem {
      * The current process's host filesystem. Use this instance directly, or dependency inject a
      * [Filesystem] to make code testable.
      */
-    val SYSTEM: Filesystem = PLATFORM_FILESYSTEM
+    val SYSTEM: Filesystem
+    get() {
+      return PLATFORM_FILESYSTEM
+    }
```

Comment:
There's something fishy here... Not using the getter syntax triggers a NullPointerException in CI. Haven't been able to reproduce locally